pull_request_template.md

```diff
@@ -1,5 +1,16 @@
 Link to my deployed project:
+Frontend: https://youthful-liskov-e3177b.netlify.app/
+Backend: https://siri-anna-auth-backend.herokuapp.com
 
 My reflections on how this week's project turned out:
 
+- Overall it went well and it was fun to do our first real fullstack application!
+- It was great to refresh our knowledge on so many different things, such as React Router, Redux Store.
+- Really good to learn about authorization and how to create logins etc
+
 Things I'd like to have clarified or explained in more detail:
+
+- As mentioned in the README we do still have one error which we don't know exactly what it means. Would be great to get some more explanation on this :)
+
+"index.js:1 Warning: Can't perform a React state update on an unmounted component. This is a no-op, but it indicates a memory leak in your application. To fix, cancel all subscriptions and asynchronous tasks in a useEffect cleanup function.
+in LogIn (at App.js:43)"
```

Comment:
To fix it you need to create `_redirects` file in the `public` directory of your react project and paste it inside:
``` 
/* /index.html 200
```

![image](https://user-images.githubusercontent.com/50926159/120884425-740a1d00-c5e3-11eb-8bc9-5093b2558132.png)